examples/source-gcloud-config-simple/README.md

```diff
@@ -0,0 +1,30 @@
+# source-gcloud-config: Simple Example
+
+### Overview
+
+This example demonstrates how to imperatively run [`source-gcloud-config`] function
+to add a ConfigMap resource containing the gcloud configurations.
+
+### Before you begin
+
+1. Install [gcloud SDK](https://cloud.google.com/sdk/docs/install)
+1. [Setting defaults for gcloud commands](https://cloud.google.com/artifact-registry/docs/gcloud-defaults)
+   Suggest setting up your `project`, `region` and `zone`
+
+### Fetch the executable
+
+This package contains the `source-gcloud-config` executable.
+```shell
+$ kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/source-gcloud-config-simple
+```
+
+### Run the function
+
+```shell
+$ kpt fn eval --exec ./source-gcloud-config .
```

Comment:
This runs as an exec and works. I don't think we want to change this to an image.
I refactored the code so it can be imported by kpt as a lib as well :)